### ISSUE TITLE
Submit to F-Droid?

### ISSUE THREAD

**Issue by opusforlife** [Luca1152/gravity-box#66]
This is a great game! Since it is open source, would you be willing to include it in F-Droid?

**Comment by Luca1152** [comment]
Yes, I will include it in F-Droid in the future, but this will have to wait for now, as I have other things I am working on (I'd have to modify the game a bit so it satisfies [F-Droid's Inclusion Policy](https://f-droid.org/en/docs/Inclusion_Policy/), which would take a bit of time).

**Comment by opusforlife** [comment]
Glad to know. Thanks! :)
